op-e2e/external_erigon/main.go

```diff
@@ -160,10 +158,6 @@ func execute(binPath string, config external.Config) (*erigonSession, error) {
 	}
 	fmt.Fscanf(sess.Err, "%d", &enginePort)
 	fmt.Printf("==================    op-erigon shim got engine port %d  ==========================\n", enginePort)
-	gm.Eventually(sess.Err, time.Minute).Should(gbytes.Say("Regeneration ended"))
```

Comment:
I think we should probably look for another message that indicates startup is done -- and we can hopefully eliminate the sleep below.  But that's outside the scope of this PR, I'll take a look and submit separately.